canonical/canonical.com
# Update application FAQ

## Done
Update application FAQ
Add the meta copy doc

## QA

- Open the copy doc in the meta copy docs
- Check the code change matches the copy change.
